src/docs/marble/svg/export-to-png.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+echo "### Removing all older pngs ###";
+for f in "../png/"*.png ;
+do
+ rm "$f";
+done
+echo "### Cleared old pngs ###";
+
+echo "### Converting SVG -> PNG ###";
+for f in *.svg ;
+do
+ inkscape "$f" --export-png="../png/`basename $f .svg`.png" --without-gui;
```

Comment:
The script should be reworked to generate PNGs into `../`. For testing purposes, you should probably also accept a command line parameter to override this default and let user provide a destination directory

Reply:
I was unable to execute the script on macos, inkscape complains about not finding the SVGs... looks like absolute file path would be necessary. let me see if I can rework the script.

Reply:
Were you running the script from within `src/docs/marble/svg` directory?

Reply:
yes I was

Reply:
btw I think having all the scripts and readme in the `/marble/` root directory will be more discoverable, once we get rid of most of the PNGs there.

Reply:
Yes..